modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -906,6 +906,20 @@ RETURN elementId(a)
 ----
 ====
 
+.Using Cypher query option connectComponentsPlanner
+====
+Query::
++
+[source,cypher]
+----
+CYPHER connectComponentsPlanner=greedy MATCH (a), (b) RETURN *
+----
+Description of the returned code::
+The Cypher query option `connectComponentsPlanner` is deprecated and will be removed without a replacement.
+The product's default behavior of using a cost-based IDP search algorithm when combining sub-plans will be kept.
+For more information, see Cypher Manual -> Cypher planner.
```

Comment:
```suggestion
For more information, see link:https://neo4j.com/docs/cypher-manual/current/query-tuning/query-options/#cypher-planner[Cypher manual -> Cypher planner].
```

Reply:
Hm, I think we have said not to put links in the description (otherwise we need to keep them updated, which can be hard to maintain)? 

But maybe we could have  something like "see the Cypher planner documentation in the Cypher manual "?

Reply:
Yes, I thought that here in the docs would be fine, but feel free to remove it.

Reply:
Ah, okay I see :)  

That sounds good 👍 So here in the documentation we have the link, but in the "real" notification description we just have `For more information, see Cypher manual -> Cypher planner`?

Reply:
Yes, that's what I thought.